### ISSUE TITLE
The exec call hangs

### ISSUE THREAD

**Issue by polRk** [testcontainers/testcontainers-node#868]
**Expected Behaviour**
Immediately return result like `docker exec containerId`

**Actual Behaviour**
It gets stuck

**Testcontainer Logs**
```
  testcontainers [INFO] [52c1ac77bd17] Container is ready +0ms
  testcontainers [DEBUG] [52c1ac77bd17] Getting container by ID... +1ms
  testcontainers [DEBUG] [52c1ac77bd17] Got container by ID +0ms
  testcontainers [DEBUG] [52c1ac77bd17] Execing container with command "/opt/ydb/bin/ydbd -s grpc://127.0.0.1:2135 admin blobstorage config init --yaml-file /opt/ydb/cfg/config.yaml"... +0ms
```

**Steps to Reproduce**
This promise will never come to an end

![Image](https://github.com/user-attachments/assets/31686348-8b35-439f-bbe0-fd06fdbcd2f6)

**Environment Information**
- Operating System: macOS 
- Docker Version: Orbstack 1.8.1
- Node version: bun 1.1.32
- Testcontainers version: 10.14.0

**Comment by polRk** [comment]
> look into why the `ydbd` command is hanging.

No, this is not the case, if you run the command directly through docker exec, they do not hang and are executed instantly

**Comment by cristianrgreco** [comment]
Could you try with NodeJS instead of Bun, and Docker instead of Orbstack, and LMK if the issue persists?

**Comment by damms005** [comment]
@cristianrgreco I can confirm it only happens in Bun. Works fine when I switched back to using Node.
